src/main/flight/imu.h

```diff
@@ -53,6 +53,7 @@ typedef union {
 
 extern attitudeEulerAngles_t attitude;
 extern matrix33_t rMat;
+extern float imuAttitudeQuaternion[3]; //attitude quaternion to use in blackbox
```

Comment:
Better export whole quaternion (but save only x,y,z in BB)
```suggestion
extern quaternion imuAttitudeQuaternion; //attitude quaternion to use in blackbox
```